### PR TITLE
Update README.md using PossibleHistoryContentValue to fix mismatch types

### PR DESCRIPTION
**Pull request by ltfschoen**
### What was wrong?

mismatched type

### How was it fixed?

use type that compiles

### CODE REVIEW

File: ethportal-api/README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ async fn main() {
     assert!(result);
 
     // Call portal_historyLocalContent endpoint and deserialize to `HistoryContentValue::BlockHeaderWithProof` type
-    let result: HistoryContentValue = client.local_content(content_key).await.unwrap();
+    let result: PossibleHistoryContentValue = client.local_content(content_key).await.unwrap();
     assert_eq!(result, content_item);
```
Comment (ltfschoen) [inline ethportal-api/README.md:45]:
```suggestion
    assert_eq!(result, PossibleHistoryContentValue::ContentPresent(content_item.clone()));
```

Comment (njgheorghita) [reply ethportal-api/README.md:45]: Yup, this is also needed, though it doesn't seem like the clone is necessary

**Review by njgheorghita — COMMENTED** [review]
Thanks! Looks like you'll also need to import the `PossibleHistoryContentValue` type at the top.

**Comment by KolbyML** [comment]
Closing this PR as it is no longer relivent and the current doc is now correct

**Pull request closed without merging**